server/sonar-web/src/main/js/components/SourceViewer/SourceViewerBase.tsx

```diff
@@ -384,25 +366,46 @@ export default class SourceViewerBase extends React.PureComponent {
     const firstSourceLine = this.state.sources[0];
     this.setState({ loadingSourcesBefore: true });
     const from = Math.max(1, firstSourceLine.line - LINES);
-    this.props
-      .loadSources(this.props.component, from, firstSourceLine.line - 1, this.props.branch)
-      .then(sources => {
-        this.props.loadIssues(this.props.component, from, firstSourceLine.line - 1).then(issues => {
-          if (this.mounted) {
-            this.setState(prevState => {
-              const nextIssues = uniqBy([...issues, ...prevState.issues], issue => issue.key);
-              return {
-                issues: nextIssues,
-                issuesByLine: issuesByLine(nextIssues),
-                issueLocationsByLine: locationsByLine(nextIssues),
-                loadingSourcesBefore: false,
-                sources: [...this.computeCoverageStatus(sources), ...prevState.sources],
-                symbolsByLine: { ...prevState.symbolsByLine, ...symbolsByLine(sources) }
-              };
-            });
+    this.props.loadSources!(
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qYww6XGHKvPuQTH&open=AWHb6qYww6XGHKvPuQTH)

[ISSUE_KEY]: AWHb6qYww6XGHKvPuQTH